nestkernel/archiving_node.cpp

```diff
@@ -275,6 +275,19 @@ nest::Archiving_Node::set_status( const DictionaryDatum& d )
     clear_history();
   }
 
+
+  if ( d->known( names::update_synaptic_elements ) )
```

Comment:
"synaptic_elements_growth_param" maybe? Since whatever parameters are updated here affect the growth dynamics of various synaptic elements? It's a bit wordy, though.